performance/benchmarks/inteceptor/summary.md

```diff
@@ -0,0 +1,74 @@
+| Apache JMeter Client          | c5.large          | 2    | 4        |
+| Apache JMeter Server 01       | c5.xlarge         | 4    | 8        |
+| Apache JMeter Server 02       | c5.xlarge         | 4    | 8        |
+| AWS EKS cluster (three nodes) | c5.xlarge         | 4    | 8        |
+
+For the scenarios explained here, we limited total number of concurrent user amount to 100 (for comparison purposes).
+
+The main performance metrics:
+
+1. **Throughput**: The number of requests that the WSO2 API Choreo Connect processes during a specific time interval (e.g. per second).
+2. **Response Time**: The end-to-end latency for an operation of invoking an API. The complete distribution of response times was recorded.
+
+In addition to the above metrics, we measure the load average and several other API request related metrics.
+
+The following are the test parameters.
+
+| Test Parameter       | Description                                                     | Values                      |
+| -------------------- | --------------------------------------------------------------- | --------------------------- |
+| Scenario Name        | The name of the test scenario.                                  | Refer to the above table.   |
+| Heap Size            | The amount of memory allocated to the application               | 1024M                       |
+| Concurrent Users     | The number of users accessing the application at the same time. | 10, 50, 100, 200, 500, 1000 |
+| Message Size (Bytes) | The request payload size in Bytes.                              | 50, 1024, 10240, 102400     |
+| Back-end Delay (ms)  | The delay added by the back-end service.                        | 0                           |
+
+The duration of each test is **900 seconds**. The warm-up period is **300 seconds**.
+The measurement results are collected after the warm-up period.
+
+[AWS EKS cluster with **c5.xlarge** Amazon EC2 instances](https://aws.amazon.com/eks/?nc2=type_a) were used to deploy WSO2 Choreo Connect.
+When doing below test scenarios, `--cpus` option is provided as 1 and concurrency level for the router provided as 2. Below table includes
+configuration details relevant to the Choreo Connect deployment.
+
+|Container Name|Requesting Memory Amount (Mi)|Requesting CPU Amount (m)|Limiting Memory Amount (Mi)|Limiting CPU Amount (m)|
+|--------------|-----------------------------|-------------------------|---------------------------|-----------------------|
+|Adapter       |500                          |500                      |500                        |500                    |
+|Enforcer      |1000                         |1000                     |1000                       |1000                   |
+|Router        |500                          |1000                     |500                        |1000                   |
+|Netty backend |4096                         |2000                     |6114                       |2000                   |
+
```

Comment:
@pubudu538 shall we add the interceptor container as well?
- Interceptor
- replicas: 2
- resources
    ```yaml
    requests:
      memory: "500Mi"
      cpu: "1000m"
    limits:
      memory: "500Mi"
      cpu: "1000m"
    ```

Reply:
+1